NuRadioMC/simulation/simulation.py

```diff
@@ -1084,6 +1102,35 @@ def _get_em_had_fraction(self, inelasticity, inttype, flavor):
                 fem = 0
         return fem, fhad
 
+    def _get_distance_cut(self, shower_energy):
```

Comment:
just a small thing.  I think it would be better if the function took the slope and intercept as parameters. Then it will be clear where the parameters came from without looking into the function.

Reply:
Then it doesn't have to be even a method of the simulation class, it can be out of its scope.